arm/Microsoft.Storage/storageAccounts/.bicep/nested_fileShare.bicep

```diff
@@ -9,11 +9,11 @@ resource share 'Microsoft.Storage/storageAccounts/fileServices/shares@2019-06-01
   }
```

Comment:
Suggested
```
param fileShareObj object
...
resource fileShare
...
module fileshare_rbac
```

Reply:
This should now be fixed. Can you verify that you agree with the changes?